lib/src/extensions/int_array.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Extension methods to convert integer arrays to List<int>
+
+import 'dart:ffi';
+
+extension Uint8Helper on Pointer<Uint8> {
+  /// Creates a [List] from `Pointer<Uint8>`.
+  ///
+  /// [length] must be equal to the number of elements stored inside the
```

Comment:
Length "must not be greater than", rather than "must be equal to", I think?

Reply:
Done